webapp/src/pluginConstants/form.ts

```diff
@@ -105,6 +105,78 @@ const serviceTypeOptions: LabelValuePair[] = [
     },
 ];
 
+export const buildStatusOptions: LabelValuePair[] = [
+    {
+        value: 'Succeded',
+        label: 'Succeded',
+    },
+    {
+        value: 'PartiallySucceded',
+        label: 'Partially Succeded',
+    },
+    {
+        value: 'Failed',
+        label: 'Failed',
+    },
+    {
+        value: 'Stopped',
+        label: 'Stopped',
+    },
+    {
+        ...filterLabelValuePairAll,
```

Comment:
can we change this filterLabelValuePairAll variable name ?

Reply:
I think what we have is fine. Do you have any suggestions?

Reply:
can you put all in between the words like filterLabelPairAllValue or filterLabelPairValues. if that doesn't change the meaning

Reply:
Yes, it changes the meaning. The meaning of the name is that it's a label-value pair that contains the value "all"